pkg/cli/context.go

```diff
@@ -71,6 +71,17 @@ func (ctx *Context) Cluster() (*Cluster, error) {
 	return NewCluster(conf), nil
 }
 
+// Clusters returns the clusters.
+func (ctx *Context) Clusters() ([]*Cluster, error) {
+	confs := ctx.ConfigManager().All()
+	clusters := []*Cluster{}
```

Comment:
nit: `var clusters []*Cluster`

A nil slice is similar here but saves an unnecessary allocation.